Reflections/Week_3_Reflection.md

```diff
@@ -0,0 +1,9 @@
+**Will Pratt**
+1. What have you accomplished this week? (Please list each accomplishment, and explain)
+* When traveling between rooms I have have addressed most bugs that could come from incorrect user input. I have added a way that healing items can be used and worked on the logic for enemy skills so that they have cooldowns as well. I also created a story class that will handle the intro and story elements like outputting the title and class selection.
+2. What challenges or difficulties did you face? If you solved them, how? If not, what have you learned so far? Have you sought help or other resources?
+*  I don't think I had too much trouble but I did notice areas where I could try to write my code more efficiently and where I could turn lines of code into functions.
+3. What do you plan to accomplish in the following week? (Please list at least 3 concrete "S.M.A.R.T." goals, along with estimated number of hours to complete, or day to complete)
+* I want to create a graph that will be a visual layout for my rooms which could take 1-2 days. I want to add detail and adjust the flow of combat so it's not as bland and feels smoother which could take 1-3 days. I also want to continue working on my story class which could take 1-3 days.
+4. What resources will you use to accomplish your goals for the upcoming week? (please list out the resources)
+* I will probably use draw.io to make my graph. 
```

Comment:
Great to see this! Making a graph to detail how a bunch of rooms will connect will likely be very helpful.